scripts/test/fuzzing.py

```diff
@@ -132,6 +132,8 @@
     'string-lifting-section.wast',
     # TODO: fuzzer support for uninhabitable imported globals
     'exact-references.wast',
+    # We cannot re-instrument such code (see the pass).
+    'instrument-branch-hints.wast',
```

Comment:
If there's no guarantee that the second instrumentation works after optimizations, then wouldn't it make sense to make the fuzzer gracefully handle that case? And if we did that, this would be fine to be initial contents, right?

Reply:
I removed the second instrumentation logic - over the long weekend I realized it can't work, at least not fully (identifying the br is hard, when we might modify the br condition, say by adding an eqz or worse). I have a different idea for how to fuzz here. For now, this PR just adds the straightforward instrumentation.

Reply:
Does this still need to be in the fuzzer ignore list after those changes?

Reply:
Actually I re-added this line, as we can't fuzz this code: it uses an import that we'd need to support in all parts of the fuzzer (our interpreter, d8, etc.). For now, I'm not sure if that is even needed.